tests/conformance/conformance.go

```diff
@@ -28,8 +30,8 @@ func main() {
 
 func execute() error {
 	args := []string{}
-	args = append(args, "--container-tag", containerTag)
-	args = append(args, "--junit-output", "/tmp/sonobuoy/results/junit.xml")
+	args = append(args, "--container-tag", containerTag,
```

Comment:
```suggestion
	args = []string{"--container-tag", containerTag,
		"--junit-output", "/tmp/sonobuoy/results/junit.xml"}
```

Why not like this?

Reply:
```
args := []string{"--container-tag", containerTag, "--junit-output", "/tmp/sonobuoy/results/junit.xml"}
```

+ dropping the lines around should work, no?

Reply:
Yeah that's much better - will push the change.